README.md

```diff
@@ -29,21 +27,26 @@ $ git clone https://github.com/AcompanhaLegis/docs.git
 $ cd docs
 </code>
 
-#### 3. Build the docker image from Dockerfile
+#### 3. You must activate the venv bin, it will be activated using
 
 <code>
-$ sudo docker build -t me-adota-docs .
+$ virtualenv venv
```

Comment:
The two lines are being displayed at one. This should not happen.

Reply:
fixed